### ISSUE TITLE
Type alias of `Any` is flagged as "unknown type" by --verifytypes

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#2959]
The "--verifytypes" option in the CLI incorrectly flags the use of an explicit `Any` type if it is in the form of a type alias.

```python
Foo = Any

def f() -> Foo: ...
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.217, which I just published. It will also be included in the next release of pylance.
